### PR TITLE
Documenting 5.2 as LTS release

### PR DESCRIPTION
**Pull request by Temidayo32**
This pull request resolve this issue #11103. This documents 5.2 as an LTS release. 

_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -7,8 +7,8 @@ Changelog
  * Maintenance: Update BeautifulSoup upper bound to 4.12.x (scott-8)
 
 
-5.2 (xx.xx.xxxx) - IN DEVELOPMENT
-~~~~~~~~~~~~~~~~
+5.2 LTS (1.11.2023)
```
Comment (gasman) [inline CHANGELOG.txt:10]: Please don't fill in the release date or remove the "in development" text yet - we'll do that when the release happens.

File: docs/releases/index.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Release notes
 
    upgrading
    5.3
-   5.2
+   5.2 LTS
```
Comment (gasman) [inline docs/releases/index.rst:9]: This should be left unchanged, as it needs to match the filename `docs/releases/5.2.md`.

**Review by gasman — CHANGES_REQUESTED** [review]

File: CHANGELOG.txt (hunk 1 above)
Comment (Temidayo32) [reply CHANGELOG.txt:10]: Oh. I will correct that. Thanks

File: docs/releases/index.rst (hunk 2 above)
Comment (Temidayo32) [reply docs/releases/index.rst:9]: Oh, I didn't realize. I will correct this too.

File: CHANGELOG.txt (hunk 1 above)
Comment (Temidayo32) [reply CHANGELOG.txt:10]: @gasman I filled in the release date in the 5.2.md file (based on the 5.2 milestone). Should I change that back to unreleased?

File: docs/releases/5.2.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Wagtail 5.2 release notes - IN DEVELOPMENT
```
Comment (gasman) [inline docs/releases/5.2.md:1]: This should also be updated to include (LTS), to match how we've handled other releases (and ensure it shows in the side menu):
```suggestion
# Wagtail 5.2 (LTS) release notes - IN DEVELOPMENT
```

Comment (Temidayo32) [reply docs/releases/5.2.md:1]: Okay. Should  I remove "- IN DEVELOPMENT" here or just add LTS alone?

File: CHANGELOG.txt (hunk 1 above)
Comment (gasman) [reply CHANGELOG.txt:10]: Yes, please change that back to unreleased.

File: docs/releases/5.2.md (hunk 3 above)
Comment (gasman) [reply docs/releases/5.2.md:1]: No, leave IN DEVELOPMENT in place - we'll remove that when the release happens.

File: CHANGELOG.txt [hunk 4]
```diff
@@ -7,8 +7,8 @@ Changelog
  * Maintenance: Update BeautifulSoup upper bound to 4.12.x (scott-8)
 
 
-5.2 LTS (1.11.2023)
-~~~~~~~~~~~~~~~~~~~
+5.2 (xx.xx.xxxx) - IN DEVELOPMENT
```
Comment (gasman) [inline CHANGELOG.txt]: Sorry, looks like you've rolled this back too far now! The "LTS" _should_ be added here.

Comment (Temidayo32) [reply CHANGELOG.txt]: Oh, I will correct that.

Comment (gasman) [reply CHANGELOG.txt]: Thanks - just the underline to fix now.

Comment (Temidayo32) [reply CHANGELOG.txt]: My bad. Sorry, this has to take so much revisions. This is my first time working on a release note

**Comment by gasman** [comment]
Thanks @Temidayo32!

**Pull request merged**